analyses/cell-type-wilms-tumor-06/run.sh

```diff
@@ -0,0 +1,21 @@
+# ids defined in image for the rstudio user, if not define as such, it is not possible to login to RStudio
```

Comment:
I have not tested this (and probably could not satisfactorily without access to your system), so please test it on your end. I think it would be better to pass the password as an argument:

```suggestion
PASSWORD="$1"

# ids defined in image for the rstudio user, if not define as such, it is not possible to login to RStudio
```

I will add what I think needs to happen in line 18 for this to work.